sdk/ownership/backend.go

```diff
@@ -14,20 +14,25 @@ import (
 
 const backendName = "ownership"
 
+const storeHashKey = "ownership_hash"
+
 // Backend is the ownership backend.
 type Backend struct {
-	storeKey *cosmostypes.KVStoreKey
+	storeKey     *cosmostypes.KVStoreKey
+	storeHashKey *cosmostypes.KVStoreKey
```

Comment:
why do you add a new store?
for what i understand it's acting as a reverse index to get the ownership from a resource hash?
I'm not against this, but it's too early to implement it and you didn't implement any getter that use it..

Reply:
>  I'm not against this, but it's too early to implement it and you didn't implement any getter that use it..

Too early for what? I just didn't want to iterate over the store because it will become bottleneck very fast. Also we don't need any getter, this store is here only for internal purposes.

Also I didn't want to copy functions to get the processes list from the store. I even would get rid of CreateServeiceOnwership.

>  And that's not the purpose of this PR ;) (you already changed the ownership structure)

The purpose of this pr is to move process to cosmos-sdk and create Ownership for them and I did it, just different way.